docs/assets/css/style.scss

```diff
@@ -3,7 +3,7 @@
 ---
 
 @import
-  "minima/skins/{{ site.minima.skin | default: 'classic' }}",
+  "minima/skins/{{ site.minima.skinType | default: 'classic' }}",
```

Comment:
I think "skin" is part of the sass file skins and not referring to the human skin type of the Customer class so we probably shouldn't change the "skin" in all .scss files

Reply:
edited